cls/SourceControl/Git/Utils.cls

```diff
@@ -446,82 +446,8 @@ ClassMethod Pull(remote As %String = "origin", preview As %Boolean = 0) As %Stat
     set branchName = outStream.ReadLine(outStream.Size)
     write !, "Pulling from branch: ", branchName
 
-    set sc =  ##class(SourceControl.Git.Utils).RunGitCommandWithInput("fetch",,.errStream,.outStream, remote, branchName)
-    if (sc=1){
-        do ..PrintStreams(errStream)
-        quit sc
-    }
-
-    write !, "Fetch done"
-    write !, "Files that will be modified by git pull: "
-
-    do ##class(SourceControl.Git.Utils).RunGitCommandWithInput("diff",,.errStream,.outStream, branchName_".."_remote_"/"_branchName, "--name-status")
-    while (outStream.AtEnd = 0) {
-        set file = outStream.ReadLine()
-        set modification = ##class(SourceControl.Git.Modification).%New()
-        set modification.changeType = $piece(file, $c(9), 1)
-        set modification.externalName = $zstrip($piece(file, $c(9),2),"<W")
-        if (modification.changeType '= "A"){
-            set modification.internalName = ##class(SourceControl.Git.Utils).NameToInternalName(modification.externalName,,0)
-        }
-        else {
-            set modification.internalName = ""
-        }
-        set files($increment(files)) = modification
-        write !, ?4, modification.changeType, ?4, modification.internalName, ?4 , modification.externalName
-    }
-    if ('$data(files)) {
-        write !, ?4, "None"
-        if preview {
-            quit $$$OK
-        }
-        write !, "Already up to date."
-        quit $$$OK
-    } elseif preview {
-        quit $$$OK
-    }
-
     set sc = ..RunGitWithArgs(.errStream, .outStream, "pull", remote, branchName)
```

Comment:
This status code should be returned at the end, I am getting an error when using the Pull From Remote command from the source control menu
ERROR #5002: ObjectScript error: <COMMAND>UserAction+80^SourceControl.Git.Utils.1 *Function must return a value at Pull+4^SourceControl.Git.Utils.